sdk/storage/azqueue/service_client.go

```diff
@@ -24,11 +24,6 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azqueue/sas"
 )
 
-// ClientOptions contains the optional parameters when creating a ServiceClient or QueueClient.
```

Comment:
This is removed as was redeclared in the same package here - `sdk/storage/azqueue/queue_client.go`